tests/test_admin.py

```diff
@@ -0,0 +1,162 @@
+from django.contrib.sessions.models import Session
+from django.urls import reverse
+
+import pytest
+from pytest_django.fixtures import admin_user
+from sessionprofile.models import SessionProfile
+
+from .factories import SessionProfileFactory
+
+
+@pytest.fixture
+def session_changelist_url():
+    return reverse("admin:sessionprofile_sessionprofile_changelist")
+
+
+def test_session_profile_sanity(client, admin_user, session_changelist_url):
+
+    client.force_login(admin_user)
+    response = client.get(session_changelist_url)
+    assert response.status_code == 200
+
+    assert SessionProfile.objects.count() == 1
+
+    session = SessionProfile.objects.get()
+    assert client.session.session_key == session.session_key
+
+
+admin_user2 = admin_user
+
+
+def test_only_session_profile_of_user_shown(
```

Comment:
It might still be possible to see the detail or delete view for another users session

Reply:
Is that intended behavior? Either way would be nice to have tests that verify this (whether it is intended or not).

Reply:
It is intended on my end that you can only see you're own sessions. IDK if this is the correct message, do we want users to know other sessions exist?

I do not think it is in scope that a super admin user can see all sessions.

Reply:
I think the current message is fine, because it does not give users extra information about other user's session. Just adding tests to verify that other users cant access/delete other session is enough in my opinion

Reply:
Done

Reply:
This still needs an assertion that deleting other users sessions is not possible, but might be nicer to move that to a separate test

Reply:
Now done